demos/local-only-todolist/README.md

```diff
@@ -0,0 +1,70 @@
+# PowerSync + Supabase Flutter Local Only to Online Mode Demo: Todo List App
+
+This demo app is an extension of the Flutter Todo List App that demonstrates how to use the PowerSync SDK for Flutter in an local only way. It allows the user to use the app offline without having to sign in. After signing in data that was kept locally is synced up. For a step-by-step guide, see [here](https://docs.powersync.com/integration-guides/supabase).
+
+# Running the app
+
+Ensure you have [melos](https://melos.invertase.dev/~melos-latest/getting-started) installed.
+
+1. `cd demos/local-only-todolist`
+2. `melos prepare`
+3. `cp lib/app_config_template.dart lib/app_config.dart`
```

Comment:
Is it possible to not require this in order to get started initially? It feels like this is only for the online mode - it would be nice to be able to get the demo started without having to setup the cloud services. It's not super important though.

Reply:
Setting up the cloud services is significant friction if they aren't needed